trade-finance-manager-api/src/graphql/resolvers/query-facilities.js

```diff
@@ -0,0 +1,18 @@
+const { getAllFacilities } = require('../../v1/controllers/facility.controller');
+const { formattedNumber } = require('../../utils/number');
+// list all facilities from the database
+exports.queryAllFacilities = async () => {
+  const rawFacilities = await getAllFacilities();
+
+  const facilities = [];
+  let facility;
+  // eslint-disable-next-line no-restricted-syntax
+  for (const item of rawFacilities) {
+    facility = item.tfmFacility;
+    facility.value = `${item.tfmFacility.currency} ${formattedNumber(item.tfmFacility.value)}`;
+    facility.facilityType = item.tfmFacility.facilityType.toLowerCase();
+    facilities.push(facility);
+  }
+
+  return { tfmFacility: facilities };
```

Comment:
should this be 

```js
return { tfmFacilities facilities };
```

Reply:
unfortunately we can't use `forEach`...
forEach loop would change the order of items coming from the database. `for of` or `for in` keeps the order as it is. 
And I need the order to stay the same because the data comes sorted from the database directly